docs/migration-guide/from-older-version.md

```diff
@@ -0,0 +1,179 @@
+- [Storyshots has been removed](https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#storyshots-has-been-removed)
+- [Addons API introduced in Storybook 7 is now required](https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#new-addons-api)
+- Ecosystem updates
+  - [Webpack4 support discontinued](https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#webpack4-support-discontinued)
+  - [IE11 support discontinued](https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#modern-browser-support)
+  - [Node 18+ is now required](https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#dropping-support-for-nodejs-16)
+  - [Next.js 13.5+ is now required](https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#require-nextjs-135-and-up)
+  - [Vue 3+ is now required](https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#require-vue-3-and-up)
+  - [Angular 15+ is now required](https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#require-angular-15-and-up)
+  - [Svelte 4+ is now required](https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#require-svelte-4-and-up)
+  - [Yarn 1 is no longer supported](https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#dropping-support-for-yarn-1)
+
+If any of these changes apply to your project, please read through the linked migration notes before continuing.
+
+If any of these new requirements or changes are blockers for your project, we recommend [migrating to Storybook 7](../../release-7-6/docs/migration-guide).
+
+You may wish to read the full migration notes for [Storybook 6 to 7](https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#from-version-65x-to-700) and [Storybook 7 to 8](https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#from-version-7x-to-800) before migrating. Or you can follow the instructions below and we’ll try to take care of everything for you!
+
+## Automatic upgrade
+
+To upgrade your Storybook:
+
+<!-- prettier-ignore-start -->
+
+<CodeSnippets
+  paths={[
+    'common/storybook-upgrade.npm.js.mdx',
+    'common/storybook-upgrade.pnpm.js.mdx',
+    'common/storybook-upgrade.yarn.js.mdx'
+  ]}
+/>
+
+<!-- prettier-ignore-end -->
+
+This will:
+
+1. Determine that none of the [breaking changes](#major-breaking-changes) apply to your project
+   - If they do, you will receive instructions on how to resolve them before continuing
+2. Upgrade your Storybook dependencies to the latest version
+3. Run a collection of _automigrations_, which will:
+   - Check for common upgrade tasks
+   - Explain the necessary changes with links to more information
+   - Ask for approval, then perform the task on your behalf
+
+## Manual migrations
+
+In addition to the automated upgrades above, there are manual migrations that might be required to get Storybook 8 working in your project. We’ve tried to minimize this list to make it easier to upgrade. These include:
```

Comment:
I disagree? "to get Storybook 8 working in your project" vs. "to get Storybook 8 working on your project". The former is correct grammer.

Also we use the "in" convention in other places, e.g.:

> Installs and initializes the specified version (e.g., @latest, @8, @next) of Storybook into your project

— https://storybook.js.org/docs/api/cli-options#init

Reply:
As we're trying to convey the message that Storybook 8 needs to operate or function properly with the user's project, I'm pretty sure that it should be "on". Since this suggests compatibility or working alongside the project.

Reply:
in : into :: on : onto. We don't say to install Storybook onto your project, we say into. Thus, we should say "in" here.

Here's [another example](https://storybook.js.org/blog/storybook-7-6/) where we use "into":

> Alternatively, for a fresh install, bootstrap Storybook into an existing app with:

I'm not sure how else to explain that "in" is more grammatically and semantically correct here. 🤷‍♂️